_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw.cpp

```diff
@@ -37,6 +37,8 @@ mfxStatus MFXVideoENCODEVP9_HW::Query(mfxCoreInterface *core, mfxVideoParam *in,
     }
     else
     {
+        core->GetHWType();
```

Comment:
you forgot to assign hw type to some variable?

Reply:
yeah, this one looks weird and needs to be fixed

Reply:
This call is done this way on purpose. During this call MSDK Core is being initialized that is the mandatory step to keep working. Feel free to suggest an alternative!

Reply:
That should not go this way, core initialized during MFXInit, but not by  GetHWType, if it is working this way - you are doing something wrong.....